tools/Generator/Xamarin.Forms.Core.json

```diff
@@ -2261,6 +2292,69 @@
         {
           "name": "ItemsSource",
           "defaultValue": "null"
+        },
+        {
+          "name": "BackgroundColor",
+          "defaultValue": "Xamarin.Forms.Color.Default"
+        },
+        {
+          "name": "CancelButtonColor",
+          "defaultValue": "Xamarin.Forms.Color.Default"
+        },
+        {
+          "name": "FontAttributes",
+          "defaultValue": "Xamarin.Forms.FontAttributes.None"
+        },
+        {
+          "name": "FontFamily",
+          "defaultValue": "null"
+        },
+        {
+          "name": "FontSize",
+          "defaultValue": "-1.0",
+          "inputType": "obj",
+          "convToModel": "makeFontSize"
+        },
+        {
+          "name": "HorizontalTextAlignment",
+          "defaultValue": "Xamarin.Forms.TextAlignment.Start"
+        },
+        {
+          "name": "IsFocused",
+          "defaultValue": "false",
+          "modelType": "bool",
+          "inputType": "bool",
+          "updateCode": "updateIsFocusesd"
```

Comment:
Little typo here :)

Reply:
True. Maybe not such a good idea to work on this in the night :D